src/UI/templates/js/Core/ui.js

```diff
@@ -0,0 +1,51 @@
+var il = il || {};
+il.UI = il.UI || {};
```

Comment:
There might be a slight naming issue. I would user the following naming: 

src/UI/templates/js/Core/replace.js

Then do:
```
var il = il || {};
il.UI = il.UI || {};
il.UI.core = il.UI.core || {};
```
In the first line, since ui.js seems a rather large scope to put all in one file.